.gitlab-ci.yml

```diff
@@ -1,42 +1,29 @@
-stages:
-  - test
-
-.tests_template: &tests_template
-  before_script:
-    - pip install tox
-    - apk add build-base git
-  stage: test
-  script: tox
+default:
   tags:
     - hc-bladerunner
```

Comment:
Could you change the tag to `cloud-integrations`? The `hc-bladerunner` runners are not available anymore